src/tools/language-server/diagnostics.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {Diagnostic, DiagnosticSeverity, Range} from 'vscode-languageserver';
+
```

Comment:
Any reason not to keep all the imports in one block?

Reply:
Mild readability improvement imo. Probably not of value, will remove.